instances/instcontext/context.go

```diff
@@ -0,0 +1,22 @@
+package instcontext
```

Comment:
i like the other way of managing context keys better, have a look at https://github.com/Peripli/service-manager/blob/master/pkg/web/context_keys.go